ads/google/a4a/traffic-experiments.js

```diff
@@ -201,10 +206,13 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '2': treatmentBranchId,
     '3': delayedControlId,
     '4': delayedTreatmentBrandId,
+    '5': sfgControlId,
+    '6': sfgTreatmentId,
   };
   if (argMapping.hasOwnProperty(arg)) {
     forceExperimentBranch(win, experimentName, argMapping[arg]);
-    return true;
+    // Neither branch of SFG is eligible for A4A, unless we're an AdSense ad.
```

Comment:
Misunderstood the purpose of this function's return statement. Everything works without this change, anyway.